_includes/v19.2/sql/aggregates.md

```diff
@@ -1,160 +1,304 @@
 <table>
 <thead><tr><th>Function &rarr; Returns</th><th>Description</th></tr></thead>
 <tbody>
-<tr><td><a name="array_agg"></a><code>array_agg(arg1: <a href="bool.html">bool</a>) &rarr; <a href="bool.html">bool</a>[]</code></td><td><span class="funcdesc"><p>Aggregates the selected values into an array.</p>
+<tr><td><code>cume_dist() &rarr; <a href="float.html">float</a></code></td><td><span class="funcdesc"><p>Calculates the relative rank of the current row: (number of rows preceding or peer with current row) / (total rows).</p>
 </span></td></tr>
-<tr><td><a name="array_agg"></a><code>array_agg(arg1: <a href="bytes.html">bytes</a>) &rarr; <a href="bytes.html">bytes</a>[]</code></td><td><span class="funcdesc"><p>Aggregates the selected values into an array.</p>
```

Comment:
why are there so many blank <a> tags in 20.1 :o

Reply:
You mean the empty `[]`s? I think that's just indicating that the argument/result can be an array of values...

Reply:
Ooh Nah I meant the <a name ="func" ></a> stuff. Not a biggie. Apparently we should be using <a id> though.

Reply:
Oh, right. I'm not sure what those are about. I suspect they're their so we have anchors to link to for specific functions? @ericharmeling, @rmloveland, do you know?

Reply:
Correct - those are there so we can link directly to e.g., `array_agg` rather than make users search for the function on the [Functions and Operators](https://www.cockroachlabs.com/docs/dev/functions-and-operators.html) page.